src/configure.rs

```diff
@@ -205,6 +205,7 @@ impl Context {
     /// Removes the transport with the specified email address
     /// (i.e. [EnteredLoginParam::addr]).
     pub async fn delete_transport(&self, addr: &str) -> Result<()> {
+        let now = time();
```

Comment:
Shouldn't this be `max(time(), add_timestamp)` so that the deletion "wins" on all devices?